platform/barefoot/sonic-platform-modules-accton/wedge100bf-32qs/sonic_platform/__init__.py

```diff
@@ -0,0 +1,2 @@
+__all__ = ['chassis', 'eeprom', 'platform', 'psu', 'sfp']
```

Comment:
How does `wedge100bf-32qs/sonic_platform` differ from the original `sonic-platform-modules-bfn-montara/sonic_platform` ? Can't we have a symlink same as we do in `sonic-platform-modules-bfn-newport/sonic_platform` ? Making a copy makes it hard to maintainable in case of the changes in the original platform plugins.

Reply:
Hi akokhan,

We only modified the wedge100bf-32qs/sonic_platform/eeprom.py file, but it doesn't seems to affect the use of "show platform syseeprom" to obtain TLV eeprom data after re-review.
So we will remove wedge100bf-32qs/sonic_platform and use symlink to link sonic-platform-modules-bfn-montara/sonic_platform. 

Thanks.